components/automate-cli/pkg/verifyserver/constants/skipmessages.go

```diff
@@ -0,0 +1,13 @@
+package constants
+
+const (
+	SKIP_CERT_TEST_MESSAGE                   = "- Using Default certificates\n- No Custom certificates provided"
+	SKIP_NODE_CERT_TEST_MESSAGE              = "- Using Default certificates\n- No Custom certificates provided"
```

Comment:
"- No Custom certificates provided\n- Using Default certificates"
Can we change the structure like this? 
First the Message related to input and then how it's been handled

Reply:
Sure makes sense,  i have made the change